src/core.c/CompUnit/PrecompilationRepository.pm6

```diff
@@ -321,7 +330,7 @@ class CompUnit::PrecompilationRepository::Default does CompUnit::PrecompilationR
 
         my @result is List = $out.lines.unique;
         if $status {  # something wrong
-            self.store.unlock;
+            #self.store.unlock;
```

Comment:
commented code
Shouldn't `$bcl` be unlocked and closed here?

Reply:
Done as a LEAVE block. See most recent commits.